src/cjwt.h

```diff
@@ -56,23 +59,25 @@ typedef struct {
      *  cty
      *  crit
      */
-} jose_t;
+} cjwt_header_t;
 
 typedef struct {
-    jose_t header;
+	cjwt_header_t header;
 
     char *iss;
     char *sub;
     char *aud;
-    struct timespec exp;
```

Comment:
Why are we taking out exp (expiration), nbf (not before, i.e. start time), and iat (issued at)  ?